core-plugins/src/main/java/co/cask/hydrator/plugin/transform/ProjectionTransform.java

```diff
@@ -64,6 +65,8 @@
     "long, float, double, bytes, string). Any simple type can be converted to bytes or a string. Otherwise, a type " +
     "can only be converted to a larger type. For example, an int can be converted to a long, but a long cannot be " +
     "converted to an int.";
+    private static final String KEEP_DESC = "Comma-separated list of fields to keep. For example: " +
```

Comment:
I see based on the code below, you can have keep or drop and not both. That should be reflected in the docs here and in drop_desc